product_docs/docs/mongo_data_adapter/5/06_features_of_mongo_fdw.mdx

```diff
@@ -32,6 +34,19 @@ MongoDB Foreign Data Wrapper supports aggregate pushdown. It pushes the aggregat
 
 For more information, see [Example: Aggregate pushdown](08b_example_aggregate_pushdown).
 
+## ORDER BY pushdown
+
+MongoDB Foreign Data Wrapper supports order by push-down. If possible, push order by clause to the remote server so that we get the ordered result set from the foreign server itself. It might help us to have an efficient merge join. NULLs behavior is opposite on the MongoDB server. Thus to get an equivalent result, we can only push down ORDERBY with either ASC NULLS FIRST or DESC NULLS LAST. Moreover, as MongoDB sorts only on
```

Comment:
don't hyphenate push-down (search for other instances of it being hyphenated and fix those too)
be consistent with the capitalization of ORDER BY. For example, (not sure these changes are correct)
```suggestion
MongoDB Foreign Data Wrapper supports ORDER BY pushdown. If possible, push the ORDER BY clause to the remote server so that we get the ordered result set from the foreign server itself. It might help us to have an efficient merge join. NULLS behavior is opposite on the MongoDB server. Thus to get an equivalent result, we can only push down ORDER BY with either ASC NULLS FIRST or DESC NULLS LAST. Moreover, as MongoDB sorts only on
```

Reply:
changes done